### PR TITLE
Update readme.md for LLC compatibility

### PR DESCRIPTION
**Pull request by codebrain**
Add remarks around beta and experimental functionality in the low level client.

Suggest we wait for merge until `7.7.0`

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -88,6 +88,14 @@ Any incompatibilities between minor versions are documented against the release.
...
 
+### Low Level Client Compatibility
+
+The Elasticsearch.Net low level client will expose functionality in REST APIs that are marked as either `experimental` or `beta`. This functionality is marked as such using the `<remarks/>` XML documentation comments, which contains the stability and a description of the compatibility guarantees. Examples below:
+
+> /// <remarks>Note: Experimental within the Elasticsearch server, this functionality is experimental and may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but experimental features are not subject to the support SLA of official GA features.</remarks>
+
+> /// <remarks>Note: Beta within the Elasticsearch server, this functionality is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.</remarks>
```
Comment (russcam) [inline readme.md]: I think we ought to say something about how this affects binary compatibility too, if a user decides to use an experimental or beta feature that ends up changing in a breaking way within a major. As far as I am aware, experimental and beta APIs are never deleted within a major, but they can change in ways that would manifest as breaking changes in the LLC.

Comment (codebrain) [reply readme.md]: Agreed.

This is not 100% clear in the XML comments that we currently emit, so I think a refresh in there is in order too.

**Comment by Mpdreamz** [comment]
I think this can be merged and backported to `7.x` now thats tracking `7.x-SNAPSHOT` 👍

**Comment by codebrain** [comment]
Merged into `master` and backported to `7.x` manually.

**Pull request merged**